store/organizations.js

```diff
@@ -109,15 +187,52 @@ const $actions = {
 
     if (organizations) commit(UPDATE_CACHE, organizations.records);
 
-    return organizations;
+    return _.cloneDeep(organizations);
+  },
+
+  async getOrganization({ commit, getters }, { id, allowReadFromCache }) {
```

Comment:
Change method signature for the one specified int the trello card.
Using two parameters. not an object 
```suggestion
  async getOrganization({ commit, getters },  id, allowReadFromCache=false) {
```